### DISCUSSION TITLE
Feedback on latest release

### DISCUSSION THREAD

**Discussion by chrsnymn** [chrsnymn/redwoodjs-stripe#60] [General]
Hi there, if you've tried out the latest release. Let me know what you think!

**Comment by AmitMirgal** [comment]
hey @chrisvdm the **redwoodjs-stripe** package is pretty awesome. no doubt in that... great work 👏🏽 

Few observation my existing **redwoodjs repo** is in typescript and when i run the 
`yarn redwoodjs-stripe setup` I would prefer all the respective files generated in ts rather than js
for instance - **StripeProductCell**
![Screen Shot 2022-08-19 at 3 34 45 PM](https://user-images.githubusercontent.com/19601060/185595932-46809cd8-7791-41ef-bd2e-57c9a18a6136.png)

Also not able to create a stripe customer.
```js
// I am using the following strategy for User mapping
 <Set
   wrap={StripeProvider}
   customer={{ 
     search: isLoggedIn ? `email: "${authUser.email}"` : '',
     create: {
       email: authUser.email,
       name: authUser.name
     }
   }}
 >
   <Route path="/category" page={CategoryPage} name="category" />
   <Route path="/product" page={ProductPage} name="product" />
 </Set>
```

**versions**
yarn v1.22.18
node v16.14.2
npm v8.5.0

Please let me know if you need any other information.
Great work 👏🏽 🚀 This is super helpful alot of heavy work is done by the package. 🙏🏽
